### DISCUSSION TITLE
Limit trigger to certain subfolders

### DISCUSSION THREAD

**Discussion by protortyp** [SilentVoid13/Templater#325] [Help]
Hi. I'm using the daily notes plugin to write my notes in a subfolder called `daily_notes`. In the past I've been using espanso for some programmatic templates, but since the Android App was released I primarily work on my phone.  Templater comes in really handy for this, I could easily re-implement my stuff with it. However, I was wondering if it is possible to limit the automatic trigger to a specified subfolder, e.g. my folder `daily_notes`. I create about 2-5 pages per day that are not related to the daily notes and obviously templater inserts the the given template into each of them.

**Comment by rapatel0** [comment]
Check out this hook I made 

https://github.com/SilentVoid13/Templater/discussions/320


Unless I'm misunderstanding, You should be able to limit the execution by adding an if/else block that tests for the folder.
Example: 
```
if (tp.file.folder() =="daily_notes") {
     output == tp.include....
} else {
    //Do nothing
}
